debian/ubuntu-advantage-tools.postinst

```diff
@@ -330,6 +331,12 @@ cleanup_apt_news_flag_file() {
     fi
 }
 
+cleanup_old_motd_stamp_file() {
+    if [ -f $OLD_MOTD_ESM_ANNOUNCE_MESSAGE_FILE ]; then
+        rm -f $OLD_MOTD_ESM_ANNOUNCE_MESSAGE_FILE
```

Comment:
Myabe I am missing something, but can't we remove all of the older motd message since we only have the `motd-contract-status` one

Reply:
Good point, we should clean those up too - I'll add that

Reply:
Updated!